Python/optimizer_symbols.c

```diff
@@ -0,0 +1,332 @@
+
+#include "Python.h"
+
+#include "cpython/optimizer.h"
+#include "pycore_code.h"
+#include "pycore_frame.h"
+#include "pycore_optimizer.h"
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <stddef.h>
+
+// Flags for below.
+#define KNOWN      1 << 0
+#define TRUE_CONST 1 << 1
```

Comment:
The KNOWN and TRUE_CONST flag bits seem redundant -- we could just check whether `type` and `const_val` (respectively) are non-NULL.